### PR TITLE
Feat: Set timer resolution (1ms)

### PR DESCRIPTION
**Pull request by SnuffSocket**
Reasoning for this is the frequent use of "time.time()" which can cause unexpected behaviour, for example in "camera.py":
&ensp;Line `self.fps = 1 / (self.newft - self.prevft)` can trigger "ZeroDivisionError" if the loop runs faster than timer resolution.
&ensp;In above case we should handle the possible zero division like so:
&ensp;&ensp;`self.fps = 1 / (self.newft - self.prevft) if self.newft > self.prevft else 0`
&ensp;&ensp;Or we could use `time.perf_counter()` instead, but it's not worth ~3x the cycles?
&ensp;&ensp;_I do a similar check on my fork along with using a exponential moving average instead._

TLDR:
&ensp;Rather than hope that user has a program open that requests a higher timer resolution. Do it ourselves?

### CODE REVIEW

**Review by dfgHiatus — CHANGES_REQUESTED** [review]

**Comment by dfgHiatus** [comment]
Added some feedback to your PR! Thanks

**Review by dfgHiatus — APPROVED** [review]
